public/style.css

```diff
@@ -10,12 +11,38 @@ label {
   display: block;
 }
 
-nav a {
-  display: inline-block;
-  margin: 1em;
+.Auth-form {
```

Comment:
CSS naming convention is `kebab-case` – I'd avoid capitals though I'm not sure it really matters.